pkgs/by-name/li/linux-manual/package.nix

```diff
@@ -21,14 +21,19 @@ stdenv.mkDerivation {
   dontBuild = true;
   doInstallCheck = true;
 
-  postPatch = ''
-    # Use scripts/kernel-doc.py here, not scripts/kernel-doc because
-    # patchShebangs skips symlinks
-
-    patchShebangs --build \
-      scripts/kernel-doc.py \
-      scripts/split-man.pl
-  '';
+  postPatch =
+    let
+      scripts = lib.escapeShellArgs [
+        # Use scripts/kernel.py rather than scripts/kernel.doc because
```

Comment:
I don't understand this comment. I don't see `scripts/kernel.py` nor `scripts/kernel.doc` mentioned anywhere in this diff.